### PR TITLE
LC/AY fix requestedToFollow nodes in registration logs

### PR DESCRIPTION
**Pull request by LorenzoCianciaruso**
Currently the "node registration completed" log is logging the requestToFollow nodes as empty while they should be populated. This is the PR to fix it

### CODE REVIEW

File: registry-core/src/main/java/com/tesco/aqueduct/registry/model/NodeRegistry.java [hunk 1]
```diff
@@ -1,14 +1,13 @@
 package com.tesco.aqueduct.registry.model;
 
-import java.net.URL;
 import java.util.List;
 
 public interface NodeRegistry {
     /**
      * @param node Node to register as new or update current state
      * @return List of URL this node should currently follow
```
Comment (sebastianlavender) [inline registry-core/src/main/java/com/tesco/aqueduct/registry/model/NodeRegistry.java:8]: Comment is now incorrect as it doesn't return a list of Nodes

Comment (LorenzoCianciaruso) [reply registry-core/src/main/java/com/tesco/aqueduct/registry/model/NodeRegistry.java:8]: nice catch, gonna remove the comment

Comment (sebastianlavender) [reply registry-core/src/main/java/com/tesco/aqueduct/registry/model/NodeRegistry.java]: I think the comment should stay but be updated. For interfaces we should retain the documentation where possible and even add some in due to this repo being opensource

**Review by sebastianlavender — APPROVED** [review]
LGTM

**Pull request merged**